README.md

```diff
@@ -11,7 +11,6 @@ Workflow
 --------
 - Client Libraries repositories should be named in this format: `nexmo-[language/runtime]`
   For example: `nexmo-java`, `nexmo-php`, `nexmo-node`
-- Development should use feature branches merged into a `develop` branch.
```

Comment:
Do we want to explicitly define what our release process should be now, if not from a `develop` branch, than from feature branches? I know that's the intent, but maybe best to say so explicitly?

Reply:
If we go by "what we do now" as the standard, no one uses a develop branch except for Ruby. We can always add this back in after we have the larger discussion about actual build steps and we get all those ducks in a row. 

For now though, we'll keep doing what we always do for each SDK.

Reply:
Right, that's not what I was asking though. What I meant is should we explicitly say what it is that we do, i.e. a github work flow (feature branch merged into master)? Instead of removing this line, amending it to describe what is our actual workflow?